games/minimal/mods/test/init.lua

```diff
@@ -19,7 +19,7 @@ local function run_hpchangereason_tests(player)
 	assert(expect == nil)
 
 	expect = { a = 234, type = "set_hp", from = "mod" }
-	player:set_hp(7, { a= 234 })
+	player:set_hp(7, { a = 234 })
```

Comment:
This line is touched, so you should correct all styling mistakes done here:
```suggestion
	player:set_hp(7, {a = 234})
```
(See https://dev.minetest.net/Lua_code_style_guidelines#Spaces.)

Reply:
I agree.